### ISSUE TITLE
Great work!

### ISSUE THREAD

**Issue by robmarkcole** [grinco/HASS-coral-rest-api#1]
Great work on this, was on my todo for a long time! I've linked in https://github.com/robmarkcole/coral-pi-rest-server/#deepstack--home-assistant would be good to explore how we can merge these projects and continue active development. For example I am keen to make deployment of custom models very straightforward

**Comment by grinco** [comment]
Thanks, mate! I haven't had a lot of time to work on it recently - have been busy with work, but I'll get to it some day - will clean everything up and push the changes to your new branch.

Better custom module support is a wonderful idea. There are some with the labels contained in the metadata, a number of pre-trained models is available on tfhub. Here's an example: https://tfhub.dev/tensorflow/lite-model/mobilenet_v2_1.0_224_quantized/1/metadata/1

**Comment by grinco** [comment]
Actually nevermind, this one has the link to labels file - in any case, I ran into a number of models that didn't work.
